### PR TITLE
Log improvements

### PR DESCRIPTION
**Pull request by Dale-Koenig**

### CODE REVIEW

File: supplementary/alica_ros1/alica_ros_proxy/src/logger/AlicaRosLogger.cpp [hunk 1]
```diff
@@ -14,19 +14,19 @@ AlicaRosLogger::AlicaRosLogger(const Verbosity verbosity, const std::string& loc
...
 
 void AlicaRosLogger::log(const std::string& msg, const Verbosity verbosity, const std::string& logSpace)
 {
     std::stringstream streamName;
-    streamName << "[ALICA] "
-               << "[" << _localAgentId << "/" << _localAgentName << "]";
+    streamName << _localAgentName;
+    if (!logSpace.empty()) {
+        streamName << "." << logSpace;
+    }
```
Comment (ntfshard) [inline supplementary/alica_ros1/alica_ros_proxy/src/logger/AlicaRosLogger.cpp:29]: should it be space after this part? in previous version `]` was like a delimiter

Comment (Dale-Koenig) [reply supplementary/alica_ros1/alica_ros_proxy/src/logger/AlicaRosLogger.cpp:29]: This is matching ros namespace format

Comment (ntfshard) [reply supplementary/alica_ros1/alica_ros_proxy/src/logger/AlicaRosLogger.cpp:29]: I meant after the name, after this if block technically

Comment (ntfshard) [reply supplementary/alica_ros1/alica_ros_proxy/src/logger/AlicaRosLogger.cpp:29]: Discussed offline

**Review by ntfshard — APPROVED** [review]

**Review by bjoernschroeder — APPROVED** [review]

**Pull request merged**